common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.mediaservice.lib.collections
+
+import com.gu.mediaservice.model.Collection
+
+object CollectionsManager {
+  def onlyLatest(collections: List[Collection]): List[Collection] = {
```

Comment:
Been talking to @akash1810 about this - so thought it's worth elaborating:

When storing collections in DynamoDB - we do it as simply as append the collection to the list. This can land up with two collections, with different `actionsData` attached. i.e.
```JSON
{
    "collections": [
        {
            "pathId": "g2",
            "path": [
                "g2"
            ],
            "actionData": {
                "author": "test",
                "date": "2015-11-23T10: 48: 59.475+00: 00"
            }
        },
        {
            "pathId": "g2/24%2F7",
            "path": [
                "g2",
                "24/7"
            ],
            "actionData": {
                "author": "test",
                "date": "2015-11-23T10: 57: 48.505+00: 00"
            }
        },
        {
            "pathId": "g2",
            "path": [
                "g2"
            ],
            "actionData": {
                "author": "test",
                "date": "2015-11-23T11: 28: 24.743+00: 00"
            }
        }
    ]
}
```

When something is rendered, and what we store in the ES record would be the deduped to latest version of the list, only Dynamo contains the whole trail (which could be useful).

i.e.
```JSON
{
    "collections": [
        {
            "pathId": "g2/24%2F7",
            "path": [
                "g2",
                "24/7"
            ],
            "actionData": {
                "author": "test",
                "date": "2015-11-23T10: 57: 48.505+00: 00"
            }
        },
        {
            "pathId": "g2",
            "path": [
                "g2"
            ],
            "actionData": {
                "author": "test",
                "date": "2015-11-23T11: 28: 24.743+00: 00"
            }
        }
    ]
}
```

TL;DR
The response from the metadata editor and ES record contain the deduped version, but dynamo contains the full record.